.github/workflows/android_cmake/start.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+set -e
+
+apt-get update -y
+
+# pkg-config sqlite3 for proj compilation
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    wget unzip ccache curl ca-certificates \
+    pkg-config make binutils sqlite3 \
+    automake
+
+cd "$WORK_DIR"
+
+if test -f "$WORK_DIR/ccache.tar.gz"; then
+    echo "Restoring ccache..."
+    (cd $HOME && tar xzf "$WORK_DIR/ccache.tar.gz")
+fi
+
+# We need a recent cmake for recent NDK versions
+wget -q https://github.com/Kitware/CMake/releases/download/v3.22.3/cmake-3.22.3-linux-x86_64.tar.gz
+tar xzf cmake-3.22.3-linux-x86_64.tar.gz
+export PATH=$PWD/cmake-3.22.3-linux-x86_64/bin:$PATH
+
+# Download Android NDK
+wget -q https://dl.google.com/android/repository/android-ndk-r23b-linux.zip
+unzip -q android-ndk-r23b-linux.zip
+
+export ANDROID_NDK=$PWD/android-ndk-r23b
+export NDK_TOOLCHAIN=$ANDROID_NDK/toolchains/llvm/prebuilt/linux-x86_64
+
+ccache -M 1G
+ccache -s
+
+# build sqlite3
+wget -q https://sqlite.org/2022/sqlite-autoconf-3370200.tar.gz
+tar xzf sqlite-autoconf-3370200.tar.gz
+cd sqlite-autoconf-3370200
+CC="ccache $NDK_TOOLCHAIN/bin/aarch64-linux-android24-clang" ./configure \
+  --prefix=/tmp/install --host=x86_64-linux-android24
```

Comment:
```suggestion
CC="ccache $NDK_TOOLCHAIN/bin/aarch64-linux-android24-clang" ./configure \
  LDFLAGS="-avoid-version" --prefix=/tmp/install --host=x86_64-linux-android24
```

Reply:
maybe you just needed to deploy libsqlite3.so.0 ?

Reply:
Not possible, Android build system does not allow this - only without version soname. `LDFLAGS="-avoid-version"` like above works - produces only 1 .so file.